### PR TITLE
fix in telegram test

### PR DESCRIPTION
**Pull request by AnetaCadova**
//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/resources/features/integrations/telegram.feature [hunk 1]
```diff
@@ -18,7 +18,10 @@ Feature: Integration - Telegram to Telegram
   #
   # TODO: WHEN TESTING MANUALY, THERE ARE 3 INTEGRATIONS TO TEST, READ COMMENTS IN THIS FILE
   #
-  Background: Clean application state and prepare what is needed
+  # Use https://desktop.telegram.org/ for testing. On channel testBotSyndesis set a bot as an admin by
+  # right clicking on the bot.
+  #
+  # Background: Clean application state and prepare what is needed
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/telegram.feature]: if this is intentionally commented out, you can remove it completely

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**